sections/shared/components/GasPriceSelect/GasPriceSelect.tsx

```diff
@@ -22,8 +24,8 @@ const GasPriceSelect: FC<GasPriceSelectProps> = ({ gasPrices, transactionFee, ..
 	const { t } = useTranslation();
 	const gasSpeed = useRecoilValue(gasSpeedState);
 	const customGasPrice = useRecoilValue(customGasPriceState);
-	const isMainnet = useRecoilValue(isMainnetState);
-	const isL2 = useRecoilValue(isL2State);
+	const isL2 = useIsL2();
+	const isMainnet = useIsL1();
```

Comment:
Is this correct as mainnet can be l2's and useIsL1 returns Goerli testnet or Eth mainnet it seems?

Reply:
This is a combination of the old naming convention. https://github.com/Kwenta/kwenta/blob/dev/store/wallet/index.ts#L25-L28

I tried to combine the following four states into two hooks to check the network id.
`isMainnetState => L1 Mainnet`
`isL1KovanState => L1 Kovan` (never used)
`isL2State => L2 Mainnet` 
`isL2KovanState => L2 Kovan` (never used)